PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,3 +22,4 @@ Platform Specific Features:
 [Optional Runtime guard:]
 [Optional Fixes #Issue]
 [Optional Deprecated:]
+[Optional API Considerations:]
```

Comment:
Can you GitHub Markdown link "API considerations" to `api/review_checklist.md`? Most folks will not read `CONTRIBUTING.md` line-by-line, so this is another opportunity to remind.

Reply:
Done.